Packs/Base/Scripts/DBotPredictURLPhishing/DBotPredictURLPhishing.py

```diff
@@ -0,0 +1,585 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import urllib
+from typing import List, Dict, Tuple
+import pandas as pd
+import base64
+import requests
+import dill
+import copy
+import numpy as np
+from tldextract import TLDExtract
+requests.packages.urllib3.disable_warnings()
+
+dill.settings['recurse'] = True
+
+NO_FETCH_EXTRACT = TLDExtract(suffix_list_urls=None, cache_dir=False)
```

Comment:
please use non capital letters - usually all capital letters is for consts so it's bit confusing